src/ArrayCache.php

```diff
@@ -67,7 +67,7 @@ public function getMultiple($keys, $default = null): iterable
         $this->validateKeys($keys);
         $results = [];
         foreach ($keys as $key) {
-            $value = $this->get($key, $default);
+            $value = $this->get((string)$key, $default);
```

Comment:
This is unnecessary - `validateKeys()` already ensures that all keys are strings.

Reply:
Yes, but it can’t be verified by static analyser. The other solution is to just suppress this issue.

Reply:
What about `assert()` or `/* @var $key string */`?

Reply:
I’ll try.